doc/en/components/data-grid.md

```diff
@@ -10,6 +10,9 @@ The $ProductName$ Data Table / Data Grid is a tabular $Platform$ grid component
 
 ## $Platform$ Data Grid Example
 
+This demo implements some of the features that are available in the Grid:
+
+Filtering, Grouping, Pin/Unpin columns, Reposition columns, Sorting, and Summaries
 
```

Comment:
can you remove line brake after "the Grid:" so that we have: 
"... the Grid: Filtering, Grouping, Pin/Unpin columns, Reposition columns, Sorting, and Summaries"

Reply:
see commit 3